roles/openshift_node/tasks/bootstrap.yml

```diff
@@ -17,10 +17,13 @@
       [Unit]
       After=cloud-init.service
 
-- name: update the sysconfig to have KUBECONFIG
+- name: update the sysconfig to have necessary variables
   lineinfile:
     dest: "/etc/sysconfig/{{ openshift_service_type }}-node"
-    line: "KUBECONFIG=/root/csr_kubeconfig"
+    line: "{{ item.line }}"
+    regexp: "{{ item.regexp }}"
+  with_items:
+  - line: "KUBECONFIG=/root/openshift_bootstrap/csr_kubeconfig"
```

Comment:
Would prefer to use `/etc/{origin|openshift}/node/bootstrap-node.kubeconfig` if all possible.

Reply:
Ok, i'll move the kubeconfig.

Reply:
Does the package name change to origin or openshift?  I've only ever seen origin whether it be enterprise or not.